src/components/language/language.service.ts

```diff
@@ -555,8 +536,8 @@ export class LanguageService {
       items: items,
       total: items.length,
       hasMore: false,
-      canCreate: !!permission?.canReadLocationCreate,
-      canRead: !!permission?.canReadLocationRead,
+      canCreate: canCreateLocation,
+      canRead: canCreateLocation,
```

Comment:
@CarsonF are you fine with doing it like this for now?

Reply:
Mmm I don't think so. I think it's important for most people to be able to read and I imagine not many people will be able to create/edit these locations

Reply:
@michaelmarshall @CarsonF so can I just set the `canRead` to `true` and remove the `CreateLocation` power while creating a user? and the `CreateLocation` power can be granted from root user like `Organization` and `Language`.

Reply:
Hmm `canCreate` really means `canAdd` in this case. "Can the current user add a location to this specific language?"

The `CreateLocation` power is completely separate. That should only apply when you are creating a location.

Reply:
@CarsonF @sethmcknight We need the business rule to determine if a user can add a location to a language. Thoughts?

Reply:
Well maybe. "Business rule" is really vague. Typically when I've said it in the past I've meant event handlers that have "reaction" logic - like syncing budget records. In my mind this just needs to be in the authorization system some how.